Plugins/DeleteUnusedBranches/GitBranchOutputCommandParser.cs

```diff
@@ -9,11 +9,12 @@ public IEnumerable<string> GetBranchNames(string commandOutput)
                 yield break;
             }
 
-            foreach (string line in commandOutput.Split('\n').Where(line => !string.IsNullOrWhiteSpace(line)))
+            foreach (string line in commandOutput.Split('\n').Where(line => !string.IsNullOrWhiteSpace(line) && line.Length > 2))
             {
-                string branchName = line.Trim('*', ' ', '\n', '\r');
+                // Removing first 2 chars: 1st is '*' for current branch or '+' for a worktree branch followed by a space
+                string branchName = line.Substring(2).Trim(' ', '\n', '\r');
```

Comment:
range operator, most calls are changed to this
```suggestion
                string branchName = line[2..].Trim(' ', '\n', '\r');
```